src/hotspot/share/opto/multnode.hpp

```diff
@@ -105,4 +106,30 @@ class ProjNode : public Node {
   ProjNode* other_if_proj() const;
 };
 
+//------------------------------TupleNode---------------------------------------
```

Comment:
These lines are rather unnecessary. More useful would be a description of what this Tuple is, and what it is used for. Are there any invariants about it? When do we expect them to appear or disappear?